case-studies/eshoppen.md

```diff
@@ -197,9 +245,9 @@ M = 1205.52 * (1/35040) * (2/16) = 0.004305 KG =~ **4.305 gCO2e**
 
 ### SCI
 
-_The sum of the SCI calculation._
+_The sum of the SCI calculation for app server._
 
-SCI =  (E * I) + M = (0.02394 KwH * 951 gCO2e/kwH) + 4.305 gCO2e = **26.178 gCO2e**
+SCI =  (E * I) + M = (0.0656 KwH * 951 gCO2e/kwH) + 4.305 gCO2e = **66.6906 gCO2e**
```

Comment:
`M` is calculated as following in L244:

M = 1205.52 * (1/35040) * (2/16) = 0.004305 KG =~ **4.305 gCO2e**

However, result of this formula is 0.0043005 .
We should fix the value.